frontend/src/pages/dashboard/components/table/drivers-table/columns/columns.ts

```diff
@@ -0,0 +1,34 @@
+import { type ColumnDef } from '@tanstack/react-table';
+import { getFullName } from 'shared/build';
+
+import { type DriverWithUserData } from '~/libs/types/types.js';
+
+const columns: ColumnDef<DriverWithUserData>[] = [
+  {
+    header: 'Full Name',
+    accessorFn: (driver: DriverWithUserData) =>
+      getFullName(driver.firstName, driver.lastName),
+    footer: 'Full name',
+    size: 250,
+  },
+  {
+    header: 'Driver License Number',
+    accessorKey: 'driver.driverLicenseNumber',
+    footer: 'Driver License Number',
+    size: 300,
+  },
+  {
+    header: 'Created At',
+    accessorFn: (driver: DriverWithUserData): string | undefined => {
+      if (!driver.driver.createdAt) {
```

Comment:
why do you added this condition?
in DriverWithUserData field driver always exists, and createdAt described as string
if you are checking for empty string, I don't see the purpose for it

Reply:
We rewrote this type many times and in some version lint gave here error without this check, but now we can remove it

Reply:
so let's do it